packages/terra-form-textarea/tests/jest/TextareaField.test.jsx

```diff
@@ -9,6 +9,12 @@ it('should render a default TextareaField component', () => {
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render a disabled TextareaField component', () => {
+  const textarea = <TextareaField disabled inputId="test-input" label="Label" />;
+  const wrapper = shallow(textarea);
+  expect(wrapper).toMatchSnapshot();
+});
+
 it('should render a TextareaField with props', () => {
```

Comment:
We need to add another jest test to verify the `inputAttrs.disabled` also renders a disabled input field